doc/source/cluster/ray-client.rst

```diff
@@ -28,6 +28,26 @@ By changing ``ray.init()`` to ``ray.client(...).connect()``, you can connect to
    #....
 
 
+You can also connect using the ClientBuilder API, but this will eventually be deprecated.
+
+.. code-block:: python
+
+   # You can run this code outside of the Ray cluster!
+   import ray
+
+   # Starting the Ray client. This connects to a remote Ray cluster. 
+   # `ray.client` will be deprecated in future releases, so we recommend
+   # using `ray.init("ray://<head_node_host>:10001")` instead.
+   ray.client("<head_node_host>:10001").connect()
+
+   # Normal Ray code follows
+   @ray.remote
+   def do_work(x):
+       return x ** x
+
+   do_work.remote(2)
+   #....
+
```

Comment:
Can we get rid of this? Let's just reference an older version of the docs instead,